pkg/kubelet/dockershim/libdocker/helpers_test.go

```diff
@@ -59,37 +59,37 @@ func TestMatchImageTagOrSHA(t *testing.T) {
 			Inspected: dockertypes.ImageInspect{
 				ID: "sha256:2208f7a29005d226d1ee33a63e33af1f47af6156c740d7d23c7948e8d282d53d",
 			},
-			Image:  "myimage@sha256:2208f7a29005d226d1ee33a63e33af1f47af6156c740d7d23c7948e8d282d53d",
+			Image:  "sha256:2208f7a29005d226d1ee33a63e33af1f47af6156c740d7d23c7948e8d282d53d",
```

Comment:
I was surprised the tests had to change for this to work... I expected *more* inputs to be accepted, and additional cases to be added, and the existing cases to stay the same

Reply:
The original test seems to be incorrect.

`myimage@sha256:2208f7a29005d226d1ee33a63e33af1f47af6156c740d7d23c7948e8d282d53d` and `sha256:2208f7a29005d226d1ee33a63e33af1f47af6156c740d7d23c7948e8d282d53d` are never the same thing ,and that is why this bug was not caught.